epp-client/src/lib.rs

```diff
@@ -139,10 +139,23 @@ pub mod extensions {
     pub mod consolidate;
     pub mod namestore;
     pub mod rgp {
+        use serde::{Deserialize, Serialize};
+
         pub mod report;
         pub mod request;
 
         pub const XMLNS: &str = "urn:ietf:params:xml:ns:rgp-1.0";
+
+        #[derive(Debug, Deserialize, Serialize)]
+        pub struct Update<T> {
+            #[serde(
+                rename = "rgp:update",
+                alias = "update",
+                alias = "upData",
+                alias = "infData"
+            )]
+            pub data: T,
+        }
```

Comment:
I think it does make sense to push this down into an `rgp`-specific file. I've found that I've mostly ended up preferring the "older" style for module nesting (`src/rgp/mod.rs` over `src/rgp.rs`).

But anyway, it seems relatively unimportant and I'm happy to accomodate how @masalachai thinks it should be done.

Reply:
I think it would be best to move it to `rgp.rs`. I also find the older module style neater for some cases and it is being used for the test module, but having this in `rgp.rs` would be nicer.

Reply:
Done.